### ISSUE TITLE
reference point for orbital forcing

### ISSUE THREAD

**Issue by christian-stepanek** [AWI-CM3/documentation#4]
Dear @JanStreffing, would it be possible to document in the HowTo of your orbit-routine the reference point for longitude of perihelion? For example, if I wanted to run mid-Holocene simulations, would the longitude of perihelion be 0.87°, based on the vernal equinox as used in PMIP and other sources, or would it rather be 180.87°, as in ECHAM?

**Comment by JanStreffing** [comment]
Yes, feel free to edit the orbital forcing part of: https://github.com/AWI-CM3/documentation_AWI-CM3/blob/master/source/how_to.rst
